js/ColorPicker/Getting-Started.md

```diff
@@ -1,20 +1,20 @@
 ---
 layout: post
-title: Getting-Started
-description: getting started
+title: Getting Started with JavaScript Color Picker | Syncfusion
+description: Learn here all about getting started with Syncfusion Essential Studio JavaScriptColor Picker control, its elements, and more details.
```

Comment:
Add space between control and platform - JavaScript Color Picker

Reply:
resolved